File: 程序员面试金典/16.2/WordsFrequency.java
class WordsFrequency {
    private Map<String, Integer> count = new HashMap<>();

    public WordsFrequency(String[] book) {
        for (String word : book)
            count.put(word, count.getOrDefault(word, 0) + 1);
    }
    
    public int get(String word) {
        return count.getOrDefault(word, 0);
    }
}

/**
 * Your WordsFrequency object will be instantiated and called as such:
 * WordsFrequency obj = new WordsFrequency(book);
 * int param_1 = obj.get(word);
 */

File: 程序员面试金典/16.2/WordsFrequency.py
from collections import Counter

class WordsFrequency:

    def __init__(self, book: List[str]):
        self.count = Counter(book)

    def get(self, word: str) -> int:
        return self.count[word]


# Your WordsFrequency object will be instantiated and called as such:
# obj = WordsFrequency(book)
# param_1 = obj.get(word)
